requirements/_base.txt

```diff
@@ -1,3 +1,3 @@
 click==3.3
 colorama==0.3.2
-psutil==2.1.2
+psutil==3.0.0
```

Comment:
This is the first version that supports ZombieProcess - bug found during manual testing.